tispark/tispark-user-guide.md

```diff
@@ -239,6 +194,58 @@ TiSparkR is a thin layer built to support the R language with TiSpark. Refer to
 
 TiSpark on PySpark is a Python package build to support the Python language with TiSpark. Refer to [this document](https://github.com/pingcap/tispark/blob/master/python/README.md) for usage.
 
+## Use TiSpark together with Hive
+TiSpark should be ok to use together with Hive. 
+You need to set environment variable HADOOP_CONF_DIR to your Hadoop's configuration folder and copy hive-site.xml to spark/conf folder before starting Spark.
+```
+val tisparkDF = spark.sql("select * from tispark_table").toDF
+tisparkDF.write.saveAsTable("hive_table") // save table to hive
+spark.sql("select * from hive_table a, tispark_table b where a.col1 = b.col1").show // join table across Hive and Tispark
+```
+
+## Load Spark Dataframe into TiDB using JDBC
+TiSpark does not provide a direct way to load data into yout TiDB cluster, but you can still load using jdbc like this:
+```scala
+import org.apache.spark.sql.execution.datasources.jdbc.JDBCOptions
+
+val customer = spark.sql("select * from customer limit 100000")
+// you might repartition source to make it balance across nodes
+// and increase concurrency
+val df = customer.repartition(32)
+df.write
+.mode(saveMode = "append")
+.format("jdbc")
+.option("driver", "com.mysql.jdbc.Driver")
+ // replace host and port as your and be sure to use rewrite batch
+.option("url", "jdbc:mysql://127.0.0.1:4000/test?rewriteBatchedStatements=true")
+.option("useSSL", "false")
+// As tested, 150 is good practice
+.option(JDBCOptions.JDBC_BATCH_INSERT_SIZE, 150)
+.option("dbtable", s"cust_test_select") // database name and table name here
+.option("isolationLevel", "NONE") // recommended to set isolationLevel to NONE if you have a large DF to load.
+.option("user", "root") // TiDB user here
+.save()
+``` 
+It is recommended to set `isolationLevel` to `NONE` to avoid large single transactions which may potentialy lead to TiDB OOM.
+
+## Statistics information
+TiSpark could use TiDB's statistic information for 
+
+1. Determining which index to ues in your query plan with the estimated lowest cost.
+2. Small table broadcasting, which enables efficient broadcast join.
+
+If you would like TiSpark to use statistic information, first you need to make sure that concerning tables have already been analyzed. Read more about how to analyze tables [here](https://github.com/pingcap/docs/blob/master/sql/statistics.md).
```

Comment:
```suggestion
If you would like TiSpark to use statistic information, first you need to make sure that concerning tables have already been analyzed. Read more about how to analyze tables [here](../sql/statistics.md).
```

Reply:
Use relative path for those files in the same repo, to make it friendly to official website users.